### PR TITLE
Add Playwright UI tests

### PR DESCRIPTION
**Pull request by mynguyenlgg**

### CODE REVIEW

File: ui-test/src/main/java/com/ui/browser/ChromeBrowserConfig.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.ui.factory.pattern;
```
[the remark this answers is not in the document]
Comment (mynguyenlgg) [reply ui-test/src/main/java/com/ui/browser/ChromeBrowserConfig.java]: I updated all comments. Please help me check it again.

File: ui-test/src/main/java/com/ui/browser/ChromeBrowserConfig.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.ui.browser;
+
+import com.microsoft.playwright.Browser;
+import com.microsoft.playwright.BrowserType;
+
+public class ChromeBrowserConfig extends AbstractBrowserManager implements BrowserConfig {
+    @Override
```
Comment (phamthai1912) [inline ui-test/src/main/java/com/ui/browser/ChromeBrowserConfig.java:7]: I think this annotation is redundant here.

File: ui-test/src/main/java/com/ui/browser/FirefoxBrowserConfig.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package com.ui.browser;
+
+import com.microsoft.playwright.Browser;
+
+public class FirefoxBrowserConfig extends AbstractBrowserManager implements BrowserConfig {
+    @Override
```
Comment (phamthai1912) [inline ui-test/src/main/java/com/ui/browser/FirefoxBrowserConfig.java:6]: same here

File: ui-test/src/main/java/com/ui/browser/AbstractBrowserManager.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+import com.microsoft.playwright.BrowserType;
+import com.microsoft.playwright.Playwright;
+
+public abstract class AbstractBrowserManager {
+    protected final Playwright playwright = Playwright.create();
+
+    protected BrowserType.LaunchOptions getLaunchOptions() {
+        return new BrowserType.LaunchOptions()
+                .setHeadless(false);
```
Comment (phamthai1912) [inline ui-test/src/main/java/com/ui/browser/AbstractBrowserManager.java:11]: setArgs to `--start-maximized` to launching browser with fullsize

File: ui-test/src/test/java/com/ui/BaseTest.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+public class BaseTest {
+    protected Browser browser;
+    protected BrowserContext browserContext;
+
+    @BeforeClass(alwaysRun = true)
+    public void beforeTestBase() {
+        browser = Configuration.getInstance().createBrowser();
+        browserContext = browser.newContext(new Browser.NewContextOptions()
+                .setViewportSize(1600, 1080));
```
Comment (phamthai1912) [inline ui-test/src/test/java/com/ui/BaseTest.java:16]: setviewportsize to null

File: ui-test/src/main/java/com/ui/browser/AbstractBrowserManager.java (hunk 4 above)
Comment (mynguyenlgg) [reply ui-test/src/main/java/com/ui/browser/AbstractBrowserManager.java]: I updated

File: ui-test/src/test/java/com/ui/BaseTest.java (hunk 5 above)
Comment (mynguyenlgg) [reply ui-test/src/test/java/com/ui/BaseTest.java]: I updated

**Review by phamthai1912 — APPROVED** [review]
LGTM

**Pull request merged**